scripts/scraper-ng/index.js

```diff
@@ -0,0 +1,58 @@
+const rehype = require("rehype");
+const reporter = require("vfile-reporter");
+
+const toVFile = require("./url-to-vfile");
+
+const examplePage = "https://developer.mozilla.org/…/div";
+const examplePath = "https://developer.mozilla.org/…/div";
+const secondExample = "https://developer.mozilla.org/…/span";
+
+const argv = require("yargs")
+  .usage("Usage: $0 <url> …")
+  .example(`$0 ${examplePage}`, "scrape a page")
+  .example(`$0 -n ${examplePath}`, "dry run (lint)")
+  .example(`$0 -n ${examplePath} ${secondExample}`, "scrape many URLs")
+  .demandCommand(1, "Not enough URLs")
+  .describe("n", "Dry run (lint)")
+  .alias("n", "dry-run")
+  .describe("q", "Suppress success messages")
+  .alias("q", "quiet")
+  .describe("v", "Expand linter notes, if applicable")
+  .alias("v", "verbose")
+  .boolean(["n", "q", "v"]).argv;
+
+const processor = rehype()
+  .use([require("./plugins/kumascript-macros")])
+  .use([
+    require("./rules/html-has-bcd-table"),
```

Comment:
I think you renamed this at some point but not here.
```suggestion
    require("./rules/html-require-compat-macro"),
```